docs/proposals/dynamic-app-parameters.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Dynamic Application Parameters
+authors:
+  - "@jannfis" # Authors' github accounts here.
+sponsors:
+  - TBD        # List all interested parties here.
+reviewers:
+  - TBD
+approvers:
+  - TBD
+
+creation-date: 2023-01-17
+last-updated: 2023-01-17
+---
+
+# Dynamic application parameters
+
+## Open questions
+
+* Should the value of a dynamic parameter be visible to a user who has `get` permissions of an Application (see also Security Implications below)?
```

Comment:
I think "yes," but don't show it in the UI if it came from a Secret. The user could always just write the value to a ConfigMap and look at it that way.

Reply:
Yep, that's a great suggestion. Thanks.